packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -420,6 +427,25 @@ contract SystemConfig is OwnableUpgradeable, ISemver, IGasToken {
         emit ConfigUpdate(VERSION, UpdateType.GAS_LIMIT, data);
     }
 
+    /// @notice Updates the EIP-1559 parameters of the chain. Can only be called by the owner.
+    /// @param _denominator EIP-1559 base fee max change denominator.
+    /// @param _elasticity  EIP-1559 elasticity multiplier.
+    function setEIP1559Params(uint32 _denominator, uint32 _elasticity) external onlyOwner {
```

Comment:
This technically needs an `onlyOwner` test for reverts case

Reply:
added